### PR TITLE
fix(config): increase default timeout

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -589,7 +617,7 @@ Advanced options can be configured as a parameter to the init() method or as env
 |isEpsagonDisabled  |DISABLE_EPSAGON            |Boolean|`false`      |A flag to completely disable Epsagon (can be used for tests or locally)            |
 |ignoredKeys        |EPSAGON_IGNORED_KEYS       |Array  |-            |Array of keys names (can be string or regex) to be removed from the trace          |
 |urlPatternsToIgnore|EPSAGON_URLS_TO_IGNORE     |Array  |`[]`         |Array of URL patterns to ignore the calls                                          |
-|sendTimeout        |EPSAGON_SEND_TIMEOUT_SEC   |Float  |`0.2`        |The timeout duration in seconds to send the traces to the trace collector          |
+|sendTimeout        |EPSAGON_SEND_TIMEOUT_SEC   |Float  |`1.0`        |The timeout duration in seconds to send the traces to the trace collector          |
```
Comment (haimrait) [inline README.md:620]: why did you change default timeout?

**Review by haimrait — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ranrib) [reply README.md:620]: To support also local runs by default
